gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/ser/binary/types/sample/SamplePersonSerializerTest.java

```diff
@@ -93,7 +93,7 @@ public void readValueAndWriteValueShouldBeSymmetric() throws IOException {
             writer.writeValue(person, buffer, nullable);
             final SamplePerson actual = reader.readValue(buffer, SamplePerson.class, nullable);
 
-            assertThat(actual, new ReflectionEquals(person));
+            Assert.assertTrue(new ReflectionEquals(person).matches(actual));
```

Comment:
nit: we tend to prefer to statically import the assertion method and to avoid `assertTrue()` which provides a terrible failure message, thus we would typically prefer: `assertThat(new ReflectionEquals(person).matches(actual), is(true))`

Reply:
Oh yes. This is a side effect of the change when I upgraded mockito to 2.x so that we could use it to mock final classes. It's not required in current review since we are using powermock. I will revert it back.